lib/aws-record/record/batch_read.rb

```diff
@@ -64,7 +64,10 @@ def execute!
       def each
         return enum_for(:each) unless block_given?
 
-        @items.each { |item| yield item }
+        @items.each do |item|
+          yield item
```

Comment:
slightly inconsistent style between `yield(batch)` and `yield item`.  I think I'd lean towards calling with the parens here.

Reply:
Updated for consistency. TY!